packages/pharos/src/utils/scss/_mixins.scss

```diff
@@ -239,10 +239,17 @@
 /// @access public
 /// @group interaction
 @mixin interactive-focus {
-  &:focus {
+  &:focus-visible {
     outline: 2px solid var(--pharos-color-focus);
     outline-offset: 2px;
   }
+
+  @supports not selector(:focus-visible) {
+    &:focus {
+      outline: 2px solid var(--pharos-color-focus);
+      outline-offset: 2px;
+    }
+  }
 }
```

Comment:
This one affects buttons, links, and their friends in sidenavs, dropdown menus, and so on.